.gitignore

```diff
@@ -1,2 +1,2 @@
-/target
+**/target
```

Comment:
this shouldn't be needed, since you added the crate to the workspace it'll use the workspace target directory and you can remove the crate target directory on you fs.

Reply:
please also remove the redundant Cargo.lock

Reply:
Should be all set, reverted the `.gitignore` and removed the Cargo.lock